.github/workflows/dev-cluster-check.yml

```diff
@@ -0,0 +1,34 @@
+name: Check development cluster
+on:
+  schedule:
+    - cron: '* */3 * * *'
```

Comment:
Is this valid?
According to https://www.freeformatter.com/cron-expression-generator-quartz.html it should be something like `0 0 * ? * *`?